<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../.designer/backup/pictures/66821.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="angleSetting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>361</width>
     <height>138</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Arial&quot;;</string>
   </property>
   <property name="title">
    <string>Angle</string>
   </property>
   <widget class="QLabel" name="note">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>315</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt &quot;Arial&quot;;
font: 300 13pt &quot;Avenir&quot;;
font: 300 italic 13pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string>Please note that the angle ranges from 0 to 90 degrees.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="angleResetPushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>74</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>335</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 13pt &quot;Arial Narrow&quot;;</string>
         </property>
         <property name="text">
          <string>MIN: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minLineEdit_2">
         <property name="placeholderText">
          <string>e.g. , 0 degree</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">
font: 13pt &quot;Arial Narrow&quot;;</string>
         </property>
         <property name="text">
          <string>MAX:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxLineEdit">
         <property name="placeholderText">
          <string>e.g. , 90 degree</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="setAnglePushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>54</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>361</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="speed">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="resistanceSlider">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TickPosition::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="setSpeedPushButton">
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetSpeedPushButton">
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>pictures/66821.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>1</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;Gill Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Actuator Controller</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>361</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Run">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>pictures/play-button-6.png</normaloff>pictures/play-button-6.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Stop">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>pictures/stop.jpeg</normaloff>pictures/stop.jpeg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Direction</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ChangeDirection">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>pictures/arrows-circle-of-two-rotating-in-clockwise-direction.png</normaloff>pictures/arrows-circle-of-two-rotating-in-clockwise-direction.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>419</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="select">
      <property name="text">
       <string>Select Port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="selectPort_comboBox">
      <item>
       <property name="text">
        <string>/dev/cu.BLTH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>/dev/cu.Bluetooth-Incoming-Port</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Connect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>280</y>
     <width>361</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="numberCycle">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Number of cycles    </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="numCycleLineEdit">
      <property name="placeholderText">
       <string>e.g. , 2 cycles</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cycleSetPushButton">
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;Al Nile&quot;;</string>
   </property>
   <property name="text">
    <string>Battery level:</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pictures/icon.qrc"/>
  <include location="pictures/directionBoarder.qrc"/>
 </resources>
 <connections/>
</ui>
